packages/terra-badge/tests/jest/__snapshots__/Badge.test.jsx.snap

```diff
@@ -1,20 +1,20 @@
 exports[`test should render a badge component in the order, icon and text with medium size 1`] = `
 <span
-  className="terra-Badge terra-Badge--medium terra-Badge--default">
+  className="badge medium default">
   <img
     alt="Test icon" />
   <span
-    className="terra-Badge-text">
+    className="text">
```

Comment:
Oh I see what you mean. I feel like what you have is totally fine then. Out of curiosity, will `-terra-badge-lib- alway be prepended? Is this something the CSS Modules does? 

> _-terra-badge-lib-___Badge__badge___8Kz6n _-terra-badge-lib-___Badge__small___9T8V4 _-terra-badge-lib-___Badge__positive___16N_U